### ISSUE TITLE
Cannot compile DropTable and CreateTable queries

### ISSUE THREAD

**Issue by abogushov** [CanopyTax/asyncpgsa#93]
Hello!

I try to create table via this code:

```python
import asyncio

from asyncpgsa import pg
import sqlalchemy as sa
from sqlalchemy.sql.ddl import CreateTable, DropTable

users = sa.Table(
    'users', sa.MetaData(),
    sa.Column('id', sa.Integer, primary_key=True),
    sa.Column('name', sa.VARCHAR(255)),
)


async def main():
    await pg.init('postgresql://localhost/test')
    await pg.query(DropTable(users))
    await pg.query(CreateTable(users))


loop = asyncio.get_event_loop()
loop.run_until_complete(main())
```

and got an error from asyncpgsa

```
Traceback (most recent call last):
  File "/project/main.py", line 20, in <module>
    loop.run_until_complete(main())
  File "/project/.venv/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "/project/main.py", line 15, in main
    await pg.query(DropTable(users))
  File "/project/.venv/lib/python3.6/site-packages/asyncpgsa/pgsingleton.py", line 65, in query
    compiled_q, compiled_args = compile_query(query)
  File "/project/.venv/lib/python3.6/site-packages/asyncpgsa/connection.py", line 60, in compile_query
    compiled_params = sorted(compiled.params.items())
AttributeError: 'NoneType' object has no attribute 'items'
```

Same code using sqlalchemy works fine:

```python
import sqlalchemy as sa
from sqlalchemy.sql.ddl import CreateTable, DropTable

users = sa.Table(
    'users', sa.MetaData(),
    sa.Column('id', sa.Integer, primary_key=True),
    sa.Column('name', sa.VARCHAR(255)),
)

engine = sa.create_engine('postgresql://localhost/test')
engine.execute(DropTable(users))
engine.execute(CreateTable(users))
```

Thanks!

**Comment by nhumrich** [comment]
I dont run migrations using asyncpg. So this has never been supported. This repo doesn't currently aupport everything sqlalchemy supports, only a subset. It looks like DropTable isnt "compiled", but rather an orm feature. 
I would accept a PR to support this, but since its a feature I dont use, I dont have the time to work on this myself.

**Comment by nhumrich** [comment]
So, looking at this error a second time, it looks like the issue might be that droptable compiled doesn't have any params. `compiled.params == None`. So, maybe this is a fixable bug. Might be worth adding a none check to connection.py line 60. Would you have time to take a look at this, and maybe add a test case for it?

**Comment by pydrive** [comment]
@nhumrich yeah, I've taken a closer look to the code of `connection.py` and decided to create my first GitHub PR ;-) #94 
Please take a look at that

**Comment by nhumrich** [comment]
This has been merged and is in version 0.25.0
